packaging/desktop/package_darwin.sh

```diff
@@ -38,6 +38,7 @@ comment=""
 keybase_binpath=${KEYBASE_BINPATH:-}
 git_remote_keybase_binpath=${GIT_REMOTE_KEYBASE_BINPATH:-}
 kbfs_binpath=${KBFS_BINPATH:-}
+redirector_binpath=${REDIRECTOR_BINPATH:-`dirname $KBFS_BINPATH`/keybase-redirector}
```

Comment:
Not sure if the naming has been discussed before. The KBFS binary is called `kbfs` on macOS. Should this be named `kbfs-redirector`?

Reply:
It hasn't been discussed, I just picked it.  I'm not sure what the best name is.  The "keybase" in the name refers to `/keybase` in my mind.  I slightly favor putting the full "keybase" in the binary name so it's obvious in the ps output where this new process came from if people notice it, and "kbfs" might not ring a bell immediately.  But I will happily entertain counter-arguments.  I think I'll merge as is for now to the epic branch, and we can decide later.

Reply:
Sounds good!